audb/core/dependencies.py

```diff
@@ -10,14 +10,51 @@
 
 
 class Dependencies:
-    r"""Hold dependencies of a database.
+    r"""Dependencies of a database.
+
+    :class:`audb.Dependencies` gathers all files
+    a database contains
+    and metadata about them
+    in a single object.
+    The metadata contain information
+    about the single files
+    like duration,
+    but also what version of the file is required.
+
+    The dependencies of a database can be requested with
+    :func:`audb.dependencies`.
+
+    Example:
+        >>> deps = Dependencies()
+        >>> deps()
+        Empty DataFrame
+        Columns: [archive, bit_depth, channels, checksum, duration, format, removed, sampling_rate, type, version]
+        Index: []
+        >>> # Request dependencies for emodb 1.1.0
+        >>> deps = audb.dependencies('emodb', version='1.1.0')
+        >>> # List all files or archives
+        >>> deps.files[:3]
+        ['db.emotion.csv', 'db.files.csv', 'wav/03a01Fa.wav']
+        >>> deps.archives[:2]
+        ['005d2b91-5317-0c80-d602-6d55f0323f8c', '014f82d8-3491-fd00-7397-c3b2ac3b2875']
+        >>> # Access a column for a given file
```

Comment:
To me that sounds like we return a column. So I would sitll prefer `Access properties for a given file` or if you want to highlight that the data is stored in columns maybe `Access a column entry for a given file`?

Reply:
Changed to `Access properties for a given file`